WordPress/gutenberg
# Icons: add new "send" icon

## What?
Adds a new "send" icon to be used by WordPress plugins that might be doing anything "sending" related, such as chats, emails, newsletters, CRM, comments, form submissions, debug messages, etc.

![Screenshot 2024-07-31 at 12 32 57](https://github.com/user-attachments/assets/b9f9df08-0576-4b96-a9f3-184b71bf9cc9)

~Pulling from [WordPress Figma design library's icon explorations](https://www.figma.com/design/e4tLacmlPuZV47l7901FEs/WordPress-Design-Library?node-id=62310-15562&t=ugMqZ2nHf89bO50G-4).~ Update: [redesigned below](https://github.com/WordPress/gutenberg/pull/64130#issuecomment-2260201259).



## Why?


"Send" is a fairly typical action in different admin and frontend UIs, and a paper plane has become a common symbol for the action. E.g. Apple Mail uses the symbol:

<img width="308" alt="Screenshot 2024-07-31 at 11 16 24" src="https://github.com/user-attachments/assets/44dda4f0-faff-4b15-b75e-2fc55c605bbe">

So does Google Gmail:

<img width="259" alt="Screenshot 2024-07-31 at 11 17 27" src="https://github.com/user-attachments/assets/f4380097-988d-4fa2-9c30-2472b1b4f2e8"><br/>

<img width="185" alt="Screenshot 2024-07-31 at 11 16 48" src="https://github.com/user-attachments/assets/f6339fd1-3057-4ccd-be5d-5fac61ebd67a">

Various chat apps also use paperplane:

Telegram
<img width="300" alt="telegram input" src="https://github.com/user-attachments/assets/2e38f77a-0d94-4149-9640-38ce8fd3af2b"/>

WhatsApp
<img width="300" alt="whatsapp input" src="https://github.com/user-attachments/assets/f57d7af0-7461-4121-99c7-9588fc23dfe9"/>

FB Messenger
<img width="300" alt="FB messenger input" src="https://github.com/user-attachments/assets/10b52a13-39e4-4ae8-8807-014f9aef771f"/>

Example usage in the plugin's feature to send blog posts as emails to yourself:

<img width="544" alt="Screenshot 2024-07-31 at 11 28 45" src="https://github.com/user-attachments/assets/725ac683-7f3c-4580-b56f-6e6a929ee7e7">

## How?


## Testing Instructions





- Run Storybook locally `npm run storybook:dev`
- Open the icon library http://localhost:50240/?path=/story/icons-icon--library
- See "send" icon
<img width="287" alt="image" src="https://github.com/user-attachments/assets/1af6301b-aaf8-48ec-91ea-a9f59e1aa0a5">



### Testing Instructions for Keyboard


## Screenshots or screencast
